cdap-common/src/main/java/co/cask/cdap/common/guice/ConfigModule.java

```diff
@@ -27,6 +28,8 @@
  */
 public final class ConfigModule extends AbstractModule {
 
+  public static final String SYSTEM_PROPERTY_PREFIX = "cdap.";
```

Comment:
Better define this in Constants. Also, maybe better to define two constants as the names for the system properties for the timeouts instead of requiring everyone to perform sting concat